cprnc_py/netcdf/scipy/netcdf.py

```diff
@@ -42,16 +43,13 @@
 import mmap as mm
 import sys
 import types
-import os
 
 import numpy as np
 from numpy.compat import asbytes, asstr
 from numpy import fromstring, dtype, empty, array, asarray
 from numpy import little_endian as LITTLE_ENDIAN
 from functools import reduce
 
-import cprnc_py.netcdf.fs_utils as fs_utils
-
 PY3 = sys.version_info[0] == 3
 
 if PY3:
```

Comment:
Rather than including this modification of scipy's netcdf.py, I feel it would be easier to maintain this snapshot if we just added a snapshot of the one additional file, scipy._lib.six. At that point, it looks like our netcdf.py would be identical to the scipy version.

Then updating the scipy netcdf.py would just be a matter of pulling in snapshots of these two files - rather than pulling in netcdf.py and merging in the necessary changes (hoping that all of those changes still apply cleanly to the new version, etc.)